met-web/src/components/survey/create/OptionsFormSkeleton.tsx

```diff
@@ -0,0 +1,39 @@
+import { Grid, Skeleton, Stack, Typography } from '@mui/material';
+import { MetPageGridContainer, RepeatedGrid } from 'components/common';
+import React from 'react';
+
+export const OptionsFormSkeleton = () => {
+    return (
+        <MetPageGridContainer container direction="row" alignItems="flex-start" justifyContent="flex-start" spacing={2}>
+            <Grid item xs={12}>
+                <Typography variant="h3">
+                    <Skeleton />
+                </Typography>
+            </Grid>
+            <Grid
+                item
+                lg={6}
+                xs={12}
+                container
+                direction="row"
+                justifyContent="flex-start"
+                alignItems="center"
+                spacing={1}
+            >
+                <Grid item xs={12}>
+                    <Typography variant="h6">
+                        <Skeleton />
```

Comment:
curious , whats this skeleton?

Reply:
It's a cool MUI component to have a loading animation. Instead of having a circular loader. Check it out here https://mui.com/material-ui/react-skeleton/